src/mongo/db/audit/audit_options.cpp

```diff
@@ -112,8 +112,11 @@ namespace mongo {
     }
 
     Status validateAuditOptions() {
-        if (!auditOptions.destination.empty() && auditOptions.destination != "file") {
-            // no validation needed if destination is not to a file
+        if (auditOptions.destination == "" || auditOptions.destination != "file") {
```

Comment:
Comparing to empty string is redundant. If value is not equal to "file" the whole expression will be true anyway